lib/init.coffee

```diff
@@ -1,13 +1,68 @@
+{BufferedProcess, CompositeDisposable} = require 'atom'
+
 module.exports =
   config:
     puppetLintExecutablePath:
-      default: ''
+      default: 'puppet-lint'
       title: 'Puppet Lint Executable Path'
       type: 'string'
     puppetLintArguments:
-      default: '--no-autoloader_layout-check'
+      default: [ '--no-autoloader_layout-check' ]
       title: 'Puppet Lint Arguments'
       type: 'string'
```

Comment:
That was actually an artifact of me switching to array's for args.  I missed removing []'s after changing it back to string.  Instead now I'm just splitting on spaces.

```
    @subscriptions.add atom.config.observe 'linter-puppet-lint.puppetLintArguments',
      (args) =>
        @args = args.split(' ')
```